doc/source/changes/version_0_26.rst.inc

```diff
@@ -44,6 +44,27 @@ Miscellaneous improvements
 * replaced `na` argument of `read_csv`, `read_excel`, `read_hdf` and `read_sas` functions by `fill_value`
   (closes :issue:`394`)
 
+* allowed to pass a label or a named group as `sheet_name` argument of the method `to_excel` and
```

Comment:
s/a label or a named group/an axis group/ ??

Reply:
The meaning of "axis group" is not obvious to me, so I guess it is not also for the users. But I can drop the 'named' since it also work for unnamed group.